easypost/shipment.py

```diff
@@ -74,6 +74,18 @@ def lowest_smartrate(self, delivery_days: int, delivery_accuracy: str) -> Rate:
 
         return lowest_smartrate
 
+    def generate_form(self, form_type, form_options={}) -> "Shipment":
```

Comment:
Please add type hints for the parameters
```suggestion
    def generate_form(self, form_type: str, form_options: dict = {}) -> "Shipment":
```

Reply:
Yeah @jchen293 can we please add type hints here per Nate's suggestion? The `dict` will need to be `Dict` and it should be `Optional[Dict[str, any]]`

Reply:
resolved